docker-32/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM conda/centos6-32-minimal-base:6.9
+MAINTAINER Michael Sarahan <msarahan@continuum.io>
+
+# locales
+RUN yum reinstall -y glibc-common \
+   && yum clean all
+
+# Set the locale
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+#RUN mkdir -p bin && \
+#  LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/concourse/concourse/releases/latest) && \
+#  LATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/') && \
+#  ARTIFACT_URL="https://github.com/concourse/concourse/releases/download/$LATEST_VERSION/fly_linux_amd64" && \
+#  curl -L ${ARTIFACT_URL} -o bin/fly && chmod +x bin/fly
+
+# This fly binary has been compiled manually
```

Comment:
Could we please document more on how this was done?  Ideally, I'd do the build in this dockerfile, but I understand that might be difficult or overly verbose.